popup.js

```diff
@@ -72,6 +72,197 @@ const containersworks = [
+            sign-ups required.
+          </p>
+          <p class="description3">
+            Exploring the future of media in Facebook's first Virtual Reality
+            app; a place to discover and enjoy 360 photos and videos on Gear VR.
+          </p>
+
+          <ul class="skills">
+            <li class="htl"><a href="#"></a>html</li>
+            <li class="css"><a href="#"></a>css</li>
+            <li class="js"><a href="#"></a>javascript</li>
+          </ul>
+          <ul class="skills2">
+            <li class="htl"><a href="#"></a>html</li>
+            <li class="js"><a href="#"></a>Ruby On rails</li>
+            <li class="css"><a href="#"></a>css</li>
+            <li class="js"><a href="#"></a>javascript</li>
+          </ul>
+          <button class="seeproject" data-modal-target="#modal1">See Project</button>
+        </div>
+      </div>
+
+      <div class="container4 desktop-reverse" id="container4" data-modal-target="#modal0">
+        <img
+          class="snapshoot"
+          src="Images/Snapshoot4.png"
+          alt="Project image"
+        />
+        <img
+          class="desktop-image"
+   
+          src="Images/Desktop-img4.png"
+          alt="Project Image"
+        />
+        <div class="daily">
+          <h2 class="Tonic2">Multi-Post Stories</h2>
+          <h2 class="Tonic4">Uber Navigation</h2>
+          <ul class="offer">
+            <li class="canopy">canopy</li>
+            <span class="point"></span>
+            <li class="backend">Back End Dev</li>
+            <span class="point"></span>
+            <li class="year">2015</li>
+          </ul>
+          <ul class="offer4">
+            <li class="canopy">Uber</li>
+            <span class="point"></span>
+            <li class="backend">Lead Developer</li>
+            <span class="point"></span>
+            <li class="year">2018</li>
+          </ul>
+          <p class="description">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <p class="description4">
+            A smart assistant to make driving more safe, efficient, and fun by
+            unlocking your most expensive computer: your car.
+          </p>
+
+          <ul class="skills">
+            <li class="htl"><a href="#"></a>html</li>
+            <li class="css"><a href="#"></a>css</li>
+            <li class="js"><a href="#"></a>javascript</li>
+          </ul>
+          <ul class="skills2">
+            <li class="htl"><a href="#"></a>html</li>
+            <li class="js"><a href="#"></a>Ruby On rails</li>
+            <li class="css"><a href="#"></a>css</li>
+            <li class="js"><a href="#"></a>javascript</li>
+          </ul>
+          <button class="seeproject" data-modal-target="#modal0">See Project</button>
+        </div>
+      </div>
+`;
+
 const modal0 = document.querySelector('.modal0');
```

Comment:
I see you're doing some work 👍🏽 but the popup is still presenting the same overflow issue pointed out by the previous reviewers, kindly look to address that 👇🏽 

![Screenshot 2023-04-16 at 00 13 43](https://user-images.githubusercontent.com/45739963/232266065-0dcfaeb0-6422-471d-8937-072122a9132e.png)

Learn more about [overflow](https://www.w3schools.com/css/css_overflow.asp) 🤓 📖